src/popup/components/TransferSendForm.vue

```diff
@@ -371,7 +371,11 @@ export default defineComponent({
       payload,
       token,
     }: Dictionary) {
-      formModel.value.selectedAsset = availableTokens.value[token] || aeternityCoin.value;
+      if (token) {
+        formModel.value.selectedAsset = availableTokens.value[token] || aeternityCoin.value;
+      } else if (!formModel.value.selectedAsset) {
+        formModel.value.selectedAsset = aeternityCoin.value;
+      }
```

Comment:
Maybe I'm missing something, but this `else` block makes no sense: there is already a check for the existence of the token, and if it's falsy the AE coin is assigned.